build/image/photon/Dockerfile

```diff
@@ -9,6 +9,9 @@ RUN go build -o manager cmd/main.go
 
 FROM photon
 
+RUN tdnf -y install shadow && \
+    useradd -s /bin/bash nsx-operator
+
 COPY --from=golang-build /source/manager /usr/local/bin/
 
-ENTRYPOINT [ "/usr/local/bin/manager"]
+ENTRYPOINT ["su", "nsx-ncp", "-m", "-c", "/usr/local/bin/manager", "-nsxconfig", "/etc/nsx-ujo/ncp.ini", "-health-probe-bind-address", ":8384"]
```

Comment:
`nsx-ncp` -> nsx-operator
`-health-probe-bind-address`: Do we need this as the default port is 8384 in code.

Reply:
ok